tools/partition_algo/base.py

```diff
@@ -75,8 +75,5 @@ def load_partition_meta(meta_file):
             raise DGLError(
                 f"num_parts[{part_meta.num_parts}] should be greater than 0."
             )
-        if part_meta.algo_name not in ["random", "metis"]:
-            raise DGLError(
-                f"algo_name[{part_meta.num_parts}] is not supported."
-            )
+
```

Comment:
I think partition/assignment algo is part of the whole distributed partition pipeline, so it's better to check the algo name to make sure the target algo is really supported(that means well tested by pipeline provider). Add a new one named `range` is preferable.